### PR TITLE
Rename edge-messaging to simplex messaging

### LINKED ISSUE #10: Rename edge-messaging protocol
as discussed naming implies a graph on a higher level of abstraction which is not explicitly defined in the lower level protocol and not necessarily needed for some other possible domain reusing it.
possible name: simplex-messaging protocol

### PR DESCRIPTION
**Pull request by efim-poberezkin**
closes #10

### CODE REVIEW

File: graph-chat.md [hunk 1]
```diff
@@ -19,16 +19,16 @@ TODO
 
 ## Duplex connection
 
-Majority of chat scenarios requires duplex (bi-directional) connections between participants. Graph-chat protocol uses multiple unidirectional connections (graph edges) created on multiple edge-messaging servers to implement duplex connections.
+Majority of chat scenarios requires duplex (bi-directional) connections between participants. Graph-chat protocol uses multiple unidirectional connections (simplexes) created on multiple simplex messaging servers to implement duplex connections.
```
Comment (epoberezkin) [inline graph-chat.md]: Not sure we need to name them - can we just drop? We can just call them "simplex connections" or where it needs to be clarified in the beginning - "simplex (unidirectional) connections". Previously it was to justify the name really.

File: graph-chat.md [hunk 2]
```diff
@@ -19,16 +19,16 @@ TODO
...
 
 The process described below establishes a duplex connection between Alice and Bob that has `n` unidirectional connections `CAi` (where `1 <= i <= n`) from Bob to Alice (created by Alice on her servers) and `n` unidirectional connections `CBi` (where `1 <= i <= n`) from Alice to Bob (created by Bob on his servers).
 
 The following symbols are used below:
-- unidirectional edge-messaging connections:
-  - `CAi` - Alice's edge-messaging connection number `i` (out of `n`) allowing Bob to send messages to Alice.
-  - `CBi` - Bob's edge-messaging connection number `i` (out of `n`) allowing Alice to send messages to Bob.
+- unidirectional simplex messaging connections:
+  - `CAi` - Alice's simplex messaging connection number `i` (out of `n`) allowing Bob to send messages to Alice.
+  - `CBi` - Bob's simplex messaging connection number `i` (out of `n`) allowing Alice to send messages to Bob.
```
Comment (epoberezkin) [inline graph-chat.md]: we can just call them "simplex connection"

File: graph-chat.md [hunk 3]
```diff
@@ -54,26 +54,26 @@ The following symbols are used below:
 To create a duplex connection initiated by Alice, Alice's and Bob's apps follow these steps:
 
 1. Alice's app initiates duplex connection:
-   1. it creates `n` unidirectional connections ("edges") `CAi` (step 1 in [edge-messaging][3]) that are defined by:
+   1. it creates `n` unidirectional connections (simplexes) `CAi` (step 1 in [simplex messaging][3]) that are defined by:
```
Comment (epoberezkin) [inline graph-chat.md]: same as above

File: readme.md [hunk 4]
```diff
@@ -101,19 +101,19 @@ Client apps should provide the following features:
 
 The chat system design is based on 2 protocols, each with the generic part, describing protocol flow and logic, and implementation part, describing protocol transports, data structures and algorithms.
 
-1. [edge-messaging protocol][6] - a low level generic messaging protocol that defines establishing and using a unidirectional connection ("graph edge") between chat participants on a single server. While this protocol is designed to support graph-chat client protocol (below), it can be used for other messaging scenarios, not limited to chats.
-2. [edge-messaging protocol implementation][7] - requirements to clients and servers implementing edge-messaging protocol, including:
+1. [simplex messaging protocol][6] - a low level generic messaging protocol that defines establishing and using a unidirectional connection (simplex) between chat participants on a single server. While this protocol is designed to support graph-chat client protocol (below), it can be used for other messaging scenarios, not limited to chats.
```
Comment (epoberezkin) [inline readme.md]: same as above - suggest dropping "simplex" as noun

File: readme.md [hunk 5]
```diff
@@ -101,19 +101,19 @@ Client apps should provide the following features:
...
+2. [simplex messaging protocol implementation][7] - requirements to clients and servers implementing simplex messaging protocol, including:
    - cryptographic algorithms to sign/verify requests and to encrypt/decrypt messages.
    - privacy requirements to the servers.
    - REST API for connections and messages.
    - WebSocket API to subscribe to connections and receive new messages.
-   - other requirements for edge-messaging servers.
-3. [graph-chat protocol][8] - a high level generic chat protocol for client applications (graph vertices) that communicate via connections (graph edges) created using edge-messaging protocol. This protocol defines connection and message types and semantics for:
+   - other requirements for simplex messaging servers.
+3. [graph-chat protocol][8] - a high level generic chat protocol for client applications (graph vertices) that communicate via connections (simplexes) created using simplex messaging protocol. This protocol defines connection and message types and semantics for:
```
Comment (epoberezkin) [inline readme.md]: same

**Comment by efim-poberezkin** [comment]
> the main comment is about using "simplex" as noun - I think we can just call it "simplex connection", "simplex" as noun sounds weird here as it already has a different meaning.

agreed, I've changed wordings accordingly, also I've changed "unidirectional [simplex]" to "simplex" across text except for a few places where the concept is being first introduced

**Pull request merged**